### PR TITLE
The Dropper: Community Edition 1.0.1

### PR DESCRIPTION
**Pull request by technodono**
- lots of small tweaks to droppers
- fixed the winners balcony bug
- dropper difficulty adjusted a bit
- players now have 2 hearts cause 1 heart wasn't enough for some drops
- now invis is turn off for levels with full health... a suggestion by @CrazyisCreeps
- fixed indentation
- timing for ending text has been adjusted

### CODE REVIEW

File: The Dropper Community Edition/map.xml [hunk 1]
```diff
@@ -105,24 +108,25 @@
...
-        <cuboid id="end-stage-15" min="241,2,101" max="244,6,104"/>
+    <cuboid id="end-stage-1" min="25,2,-1" max="28,6,3"/>
+    <cuboid id="end-stage-2" min="29,7,1" max="27,11,-2"/>
+    <cuboid id="end-stage-3" min="133,2,-1" max="136,6,3"/>
+    <cuboid id="end-stage-4" min="187,2,-1" max="190,6,3"/>
+    <cuboid id="end-stage-5" min="241,2,-1" max="244,6,3"/>
+    <cuboid id="end-stage-6" min="25,2,50" max="28,6,53"/>
+    <cuboid id="end-stage-7" min="79,2,50" max="82,6,53"/>
+    <cuboid id="end-stage-8" min="133,2,50" max="136,6,53"/>
+    <cuboid id="end-stage-9" min="187,2,50" max="190,6,53"/>
```
Comment (CoWinkKeyDinkInc) [inline The Dropper Community Edition/map.xml:119]: `<cuboid/>` tags need to be indented under `<union>` properly.

File: The Dropper Community Edition/map.xml [hunk 2]
```diff
@@ -213,5 +218,8 @@
 <hunger>
     <depletion>off</depletion>
 </hunger>
+<itemkeep>
+    <item>diamond</item>
```
Comment (CoWinkKeyDinkInc) [inline The Dropper Community Edition/map.xml:222]: You said this information isn't important anymore so it should be removed.

File: The Dropper Community Edition/map.xml [hunk 3]
```diff
@@ -12,7 +12,7 @@
     <author uuid="6863869b-4b8c-4445-b778-a8e016775ae4" contribution="Participant"/> <!-- Cazadorian -->
     <author uuid="a9f11454-beac-4cc3-a138-1780802c8794" contribution="Participant"/> <!-- Crazy_ -->
     <author uuid="fe3608b7-d105-4029-8800-34b3147065b6" contribution="Participant"/> <!-- rockymine -->
-    <author uuid="b1d3f382-6317-4c34-bda2-1aefbed6c8cd" contribution="Participant"/> <!-- _CherryPie_ -->
+    <author uuid="b1d3f382-6317-4c34-bda2-1aefbed6c8cd" contribution="Participant"/> <!-- _cherrypie_ -->
```
Comment (CoWinkKeyDinkInc) [inline The Dropper Community Edition/map.xml:15]: Username is spelled `_CherryPie_`.

File: The Dropper Community Edition/map.xml (hunk 1 above)
Comment (calcastor) [reply The Dropper Community Edition/map.xml:119]: .. why remove permanent invisibility?

**Comment by CoWinkKeyDinkInc** [comment]
Thanks for the fixes!

**Pull request merged**